src/routes/wallets.js

```diff
@@ -0,0 +1,11 @@
+var express = require('express'),
+    wallets = require('../api').wallets;
+
+
+module.exports = express.Router()
+  .post('/', function(req, res, next) {
+    // TODO validate request body
+    wallets.create(req.app.get('conf'), req.body)
+      .then(function() { res.send({}); })
```

Comment:
I can't think of any useful fields for us to put in this response. `success: true` seems redundant, we know the request was successful because of the status code. The api doesn't need to give back any wallet details to the client, I don't think.